tests/unit/s2n_tls13_server_finished_test.c

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+
+#include "tls/s2n_connection.h"
+#include "tls/s2n_tls.h"
+#include "stuffer/s2n_stuffer.h"
+#include "utils/s2n_safety.h"
+
+static int reset_stuffers(struct s2n_stuffer *reread, struct s2n_stuffer *wipe)
```

Comment:
you have this function twice - maybe move to the test library so there's one implementation and could be useful in other test cases

Reply:
i'lll prefer to hold off moving this as the function seems pretty specific to these 2 tests at this point